routes/intallatievergadering.ts

```diff
@@ -333,4 +349,272 @@ async function movePersons(installatievergaderingId: string) {
   await updateSudo(sparql);
 }
 
+async function copyCommuneMandatarisInstancesToOCMW(
+  orgaanItFrom: string,
+  orgaanItTo: string,
+) {
+  await clearMandatarisInstancesFromOrgaan(orgaanItTo);
+  await constructNewMandatarisInstances(orgaanItFrom, orgaanItTo);
+}
+
+type SimpleTriple = {
+  subject: string;
+  predicate: string;
+  object: { value: string; type: string; datatype: string };
+};
+
+async function constructNewMandatarisInstances(
+  orgaanItFrom: string,
+  orgaanItTo: string,
+) {
+  const triples = await constructNewMandatarisInstancesWithOldUris(
+    orgaanItFrom,
+    orgaanItTo,
+  );
+  const { newTriples: transformedTriples, mandatarisLinks } =
+    transformToNewMandatarisAndMembershipTriples(triples);
+
+  await insertTransformedTriples(transformedTriples);
+  await insertMandatarisLinks(mandatarisLinks);
+}
+
+async function constructNewMandatarisInstancesWithOldUris(
+  orgaanItFrom: string,
+  orgaanItTo: string,
+): Promise<SimpleTriple[]> {
+  const bestuursfunctieCodeMapping = {
+    // gemeenteraadslid -> lid raad voor maatschappelijk welzijn
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000011':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000015',
+    // voorzitter gemeenteraad -> voorzitter raad voor maatschappelijk welzijn
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000012':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000016',
+    // schepen -> lid vast bureau
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000014':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000017',
+    // aangewezen burgemeester -> voorzitter vast bureau
+    'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/7b038cc40bba10bec833ecfe6f15bc7a':
+      'http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000018',
+  };
```

Comment:
There are some constants defined in `controllers\linked-mandataris.ts`, might be a good idea to extract those and not have this duplication?

Reply:
Hmmm unfortunately the burgemeester is different ...

Reply:
good point about burgemeester being differnet, i think in the linking part it is important to do both, in the iv only the aangewezen burgemeester will count